proxy/envoy/config.go

```diff
@@ -184,13 +184,28 @@ func buildOutboundFilters(instances []*model.ServiceInstance, services []*model.
 	suffix := sharedInstanceHost(instances)
 	httpConfigs := make(HTTPRouteConfigs)
 
+	// get all the route rules applicable to the instances
+	rules := config.RouteRulesBySource("", instances)
```

Comment:
The top-level call takes instances and services, so I was just following that naming pattern. Pretend instance is always local instances throughout.